### ISSUE TITLE
radarr .title_wrapper h1

### ISSUE THREAD

**Issue by nanoymaster** [cardigann/cardigann#399]
Hi,

Sonarr works fine.
When trying to use Radarr the console floods with:
level=warning msg="Indexer \"<indexer>\" failed: Expected to find `.title_wrapper h1`"
for all indexers.

Any ideas on how I can fix this?

**Comment by lxt2** [comment]
How are you connecting Radarr to Cardigann? I use it with Torznab and it works great. What cardigann version are you on?

**Comment by nanoymaster** [comment]
yea with torznab... just updated and now 1.10.1.. same issue.
to force it can click on movie and either manual or automated search

**Comment by lxt2** [comment]
Sounds like this might be related to a specific indexer, can you disable some of them and see whether it's a specific one?

**Comment by nanoymaster** [comment]
![cardigann](https://user-images.githubusercontent.com/1678085/31603588-8ad41d76-b258-11e7-9070-49876971425b.png)

**Comment by aolwas** [comment]
Hi, 

I've got the same behavior with yggtorrent indexer (with definition file from #408): works well on sonarr but not on radarr with the .title_wrapper error

**Comment by vidschofelix** [comment]
i have the same.
You get this while searching for a specific movie, right?  
From my logs:
`Cardigann HTTP request failed: [502:BadGateway] [GET] at [http://192.168.178.32:5060/torznab/xxx/api?t=movie&cat=2000,2010,2020,2030,2035,2040,2045,2050,2060&extended=1&apikey=(removed)&offset=0&limit=100&imdbid=1396484]
`
The requests itself returns 
`
<error>
<code>900</code>
<description>Expected to find `.title_wrapper h1`</description>
</error>
`
Looking at the source of the the imdb-lookup (https://github.com/cardigann/cardigann/blob/0a246f1394de4276dcfaac7c70d2499f20e513dc/vendor/github.com/cardigann/go-imdb-scraper/scrape.go#L18) it looks he gets the raw parameter from imdbid. If if add 'tt' to the beginning of the imdbid it works. 
Looks like the api has to add this to the request for the go-imdb-scraper. I dont have enough experience in golang to do it myself, but it would be awesome if someone could add this simple fix :)

From the specs of newznabs, its correct to transmit the id without 'tt', bit the imdb scraper needs the tt in front.

TL;DR: add 'tt' to imdbid before looking up movie name.

**Comment by ldexterldesign** [comment]
@vidschofelix 

https://github.com/cardigann/go-imdb-scraper/pull/1

<img width="650" alt="screen shot 2018-03-02 at 20 58 27" src="https://user-images.githubusercontent.com/1715809/36919107-7a3707e2-1e5c-11e8-912b-6e1bcfd81c49.png">

💋
